.gitignore

```diff
@@ -39,3 +39,6 @@ tmp/
 
 # VS Code
 .vscode
+
+# PyCharm
+.idea
```

Comment:
Not really needed for the topic of the PR but no strong feelings about keeping it here or not.